openvpn-install.sh

```diff
@@ -1286,6 +1286,34 @@ function removeOpenVPN() {
 	fi
 }
 
+function listcerts () {
+# Original Script from PiVPN: list clients script
+# Modified Script to add Certificate expiration Date -- psgoundar  
+INDEX="/etc/openvpn/easy-rsa/pki/index.txt"
+printf "\\n"
+if [ ! -f "${INDEX}" ]; then
+        echo "The file: $INDEX was not found!"
+        exit 1
+fi
+printf "\\e[1m::: Certificate Status List :::\\e[0m\\n"
+printf "\\e[4mStatus\\e[0m ::  \\e[4mName\\e[0m\\e[0m        ::  \\e[4mExpiration \\e[0m\\n" 
+while read -r line || [ -n "$line" ]; do
+    STATUS=$(echo "$line" | awk '{print $1}')
+    NAME=$(echo "$line" | awk '{print $5}' | awk -FCN= '{print $2}')
+    EXPD=$(echo "$line" | awk '{if (length($2) == 15) print $2; else print "20"$2}' | cut -b 1-8 | date +"%b %d %Y" -f -)
+        
+    if [ "${STATUS}" == "V" ]; then
+         printf "     Valid   ::   %s :: %s\\n" "$NAME" "$EXPD"
+    elif [ "${STATUS}" == "R" ]; then
+        #printf "     Revoked   ::   %s :: %s\\n" "$NAME" "$EXPD"
+		continue
+    else
+        printf "     Unknown   ::   %s :: %s\\n" "$NAME" "$EXPD"
+    fi
```

Comment:
Can we use `\t` instead of multiple spaces?

Reply:
Changed to Tabs